Unix/scripts/installntlm-ubuntu16

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
```

Comment:
I'd like some comments at the very beginning on what, exactly, this script is for. I'm not asking for a book, just a few lines.

Does the script take any parameters? If so, you should have usage. But if not, I wouldn't mine actually echoing what the script does and asking if we should proceed? This scripts makes a lot of setup changes on your machine, and there's no easy way to "undo" it. As such, if this was run inadvertently, it would be nice to have a prompt of what the script does (again, just a few lines), and then ask "Proceed [No] ?", default No, take Y or YES or y or yes or whatever as affirmative.

While the prompt would be nice, minimally AT LEAST include comments as to what is going on.

Will sign off, do what you feel is right, I prefer the prompt though. Also, was this checked in as executable?